README.md

```diff
@@ -60,8 +70,7 @@ pip install -e '.[all]'
 ---
 
 # Get started
-LitGPT is a command-line tool to use, pretrain, finetune and deploy LLMs.
-
+LitGPT is CLI and config based. Select the model and the action you want to take on that model (finetune, pretrain, deploy, etc...):    
```

Comment:
```suggestion
LitGPT is CLI and config-based. Select the model and the action you want to take on that model (finetune, pretrain, deploy, evaluate etc...):    
```

Reply:
Not particularly important, but maybe `(pretrain, finetune, evaluate, deploy, etc...)`?
Better reflects development process.

Reply:
Sure, happy to update the ordering from `deploy, evaluate` to `evaluate, deploy`. Haha, in reality it's probably more like `pretrain, fintune, evaluate, deploy, evaluate, finetune, evaluate, deploy ...`

Reply:
In reality it might never make it to `deploy` stage.

Reply:
@Andrei-Aksionov if you can't deploy models with litGPT then this isn't useful to enterprises... litGPT is more than just a research tool ;)

Reply:
@williamFalcon It wasn't about LitGPT, more like "Vietnam flashbacks" from one of the projects from my previous job.